.changeset/serious-ligers-film.md

```diff
@@ -0,0 +1,18 @@
+---
+'@leafygreen-ui/checkbox': minor
+'@leafygreen-ui/emotion': minor
+'@leafygreen-ui/hooks': minor
+'@leafygreen-ui/lib': minor
```

Comment:
I believe we removed the default of `document` from `useEventListener` which would be breaking.

Reply:
It still defers defaulting to `document` in the `useEffect` (i.e. `?? document`) of `useEventListener`, so the behavior won't change there

Reply:
🤦 Missed that, makes sense!